app/assets/javascripts/components/question_tracker/master_question.js.coffee

```diff
@@ -0,0 +1,48 @@
+{div, span} = React.DOM
+
+modulejs.define 'components/question_tracker/master_question',
+  [
+    'components/question_tracker/question_summary',
+    'components/question_tracker/question_adder',
+    'components/question_tracker/open_response',
+    'components/question_tracker/image_question',
+    'components/question_tracker/multiple_choice'
+
+  ],
+  (QuestionSummaryClass, QuestionAdderClass, OpenResponseClass, ImageQuestionClass, MultipleChoiceClass) ->
+
+    QuestionSummary = React.createFactory QuestionSummaryClass
+    QuestionAdder   = React.createFactory QuestionAdderClass
+    OpenResponse    = React.createFactory OpenResponseClass
+    ImageQuestion   = React.createFactory ImageQuestionClass
+    MultipleChoice  = React.createFactory MultipleChoiceClass
+
+    React.createClass
+
+      update: (newState) ->
+        @props.update(newState)
+
+      render: ->
+        question = @props.question
+        details = question.question
+        edit = @props.edit
+        update = @update
+
+        (div {className: 'master-question'},
+          (QuestionAdder {question: question, update: update, edit: edit})
+          if @props.question
```

Comment:
You have a mix of question and @props.question and edit and @props.edit.  No bigee.